plugins/no_sir.py

```diff
@@ -0,0 +1,20 @@
+import re
+from errbot import BotPlugin
+
+
+class no_sir(BotPlugin):
+    """
+    Do not use sir
+    """
+
+    def callback_message(self, msg):
+        emots = [':D']
+        match_sir = re.search(r'\bsir\b', '\bSir\b', msg.body)
+        if match_sir:
+            self.send(
+              msg.frm,
+              '@{}, Do not use sir in your conversation. {}'.format(
```

Comment:
Please remove `comma` after mentioning the user!
Although the returned message is OK, I guess it could be more informative.
What do you think @sladyn98?

Reply:
@KVGarg Definitely if you have any suggestions you could have written here :)

Reply:
I guess we can frame a message somewhat like this
```
Hi @<username> :wave:! We don't use "Sir" in the community during the conversation. It's perfectly fine to call them via there usernames or handle. Also, If you to do such think you can better go with buddy :smiley: 
```
Something like this the message can be framed!